src/mlpack/methods/ann/layer/reparametrization_impl.hpp

```diff
@@ -83,19 +83,16 @@ void Reparametrization<InputDataType, OutputDataType>::Backward(
 
 template<typename InputDataType, typename OutputDataType>
 template<typename InputType>
-double Reparametrization<InputDataType, OutputDataType>::klForward(
-    const InputType&& input)
+inline double Reparametrization<InputDataType, OutputDataType>::KLForward(
+    const InputType&& mean, const InputType&& stdDev)
```

Comment:
Oh, I meant actually inlining these by removing `KLForward()` and placing the implementation where you've used it in `Loss()`.  Sorry, I should have been clearer. :)

Reply:
No, it's not your fault. You were clear enough. :) Later @zoq commented on that PR and even he mentioned about inlining this function. I took that as making the function inline. Though, now I am not sure what he exactly meant. @zoq Can you please clarify what you meant by inline in this case? Did you mean inlining it into the `Loss()` function?

Reply:
Sorry for the confusion, what I meant was to implement ``KLForward`` into the ``Loss`` function; that way we can avoid the extra split. Does this sound reasonable?

Reply:
I can't think of a case where the `KLForward()` and `KLBackward()` will be used besides the `Forward()` and `Backward()` functions of this layer. So, I guess you both are correct, there isn't any need to have them as seperate functions. I will inline this into the `Loss()` function.